### DISCUSSION TITLE
New User creation

### DISCUSSION THREAD

**Discussion by electric-m** [ellite/Wallos#324] [Q&A]
Hey there,

I am pretty sure I am being dumb right now, but I cannot seem to find a way to create a second user, aside from the initial user.
Any chance I am missing a point here?

Thanks
Michael

**Comment by ellite** [comment]
Hi. You're not being dumb or missing anything.
As a tool that I initially built for myself I didn't implement a way to add users.
I have it on my list since the day I released it because that was requested a lot.
I have done some work towards that. But still a long way to go.
Maybe even migrating out of SQLite.
It will be implemented eventually, but for now it's not available.
Regards.

**Reply by electric-m** [reply]
Right, thank you for answering me.
Never mind, I was just wondering, if it is possible, since I read about informing other household members in the settings.
Would indeed be nice, if I could add my wife as her own user, so we could collaborate, but wallos ist still great as it is.

**Reply by ellite** [reply]
Indeed. Household solves some of the issues, but both would need to share the same account.

Glad you enjoy it ☺️

Regards.

**Comment by ellite** [comment]
Hi. It is now possible to create new users with version 2.0.
Regards.

**Reply by electric-m** [reply]
Awesome, I’m gonna check it out!
